automerge-backend-wasm/scripts/js_tests.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+if [[ -e "../automerge" ]]
+then
+    echo "Automerge repo exists. Skipping cloning..."
+else
+    echo "Cloning automerge repo to ../../automerge"
+    git clone https://github.com/automerge/automerge.git ../../automerge
```

Comment:
Can we checkout a particular commit to make these tests reproducible?

Reply:
Done